src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/rewriter/ordinal/OrdinalRewriterRuleTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.unittest.rewriter.ordinal;
+
+import com.alibaba.druid.sql.SQLUtils;
+import com.alibaba.druid.sql.ast.expr.SQLQueryExpr;
+
+import com.amazon.opendistroforelasticsearch.sql.rewriter.matchtoterm.VerificationException;
+import com.amazon.opendistroforelasticsearch.sql.rewriter.ordinal.OrdinalRewriterRule;
+import com.amazon.opendistroforelasticsearch.sql.util.SqlParserUtils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.sql.SQLFeatureNotSupportedException;
+
+import static org.hamcrest.Matchers.containsString;
+
+/**
+ * Test cases for ordinal aliases in GROUP BY and ORDER BY
+ */
+
+public class OrdinalRewriterRuleTest {
+
```

Comment:
I have potentially funny test case 

    SELECT lastname AS 1 from bank GROUP BY 1

Reply:
Checked the MySQL standard, integers cannot be valid field alias.